xml/docu_styleguide.language.xml

```diff
@@ -405,6 +405,49 @@
    </para>
    </sect2>
 
+ <sect2 xml:id="sec-quotation">
+  <title>Quotations</title>
+  <para>
+  Use quotations to quote from sources, such as books.
+  In all other cases, do not use quotation marks:
+  </para>
+  <itemizedlist>
+  <listitem>
+  <para>
+  For computer input, computer output, and user
+  interface elements, use different markup.
+  See also <xref linkend="sec-command-line"/> and
+  <xref linkend="sec-ui-item-markup"/>.
+  </para>
+  </listitem>
+  <listitem>
+  <para>
+  Do not use quotation marks to indicate irony.
+  Avoid irony in technical writing.
+  See also <xref linkend="sec-tone"/>.
+  </para>
+  </listitem>
+  </itemizedlist>
+  <para>
+  To create quotations, use the <tag class="emptytag">quote</tag>
+  element, as it is easier to localize than hardcoded quotation marks and
+  always provides typographic quotes.
```

Comment:
This is all correct, but it could be misunderstood that the _user_ has to provide the quotation marks (which would be wrong). It's quite the opposite, the transformation provides it depending on the language.

I would slightly rephrase it:

```suggestion
  To create quotations, use the <tag class="emptytag">quote</tag>
  element. This element makes it easier to localize than hardcoded quotation marks.
  During processing, localized quotation marks are added automatically.
```

Reply:
Agree with Toms!

Reply:
OK, thanks, I'll rephrase it a bit.